Description: pass CPPFLAGS to CMAKE_CXX_FLAGS
Author: HIGUCHI Daisuke (VDR dai) <dai@debian.org>
Forwarded: not-needed
Origin: vendor
Last-Update: 2013-07-06

diff --git a/qt4/toolbar/CMakeLists.txt b/qt4/toolbar/CMakeLists.txt
index a162b05..4601afb 100644
--- a/qt4/toolbar/CMakeLists.txt
+++ b/qt4/toolbar/CMakeLists.txt
@@ -1,4 +1,5 @@
 project(plasma-uim)
+set(CMAKE_VERBOSE_MAKEFILE on)
 
 find_package(KDE4 REQUIRED)
 include(KDE4Defaults)
@@ -18,6 +19,8 @@ set(uim_SRCS
 # from Qt4 header files
 string(REGEX REPLACE "-Wconversion" ""
         CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} $ENV{CPPFLAGS}")
  
 kde4_add_plugin(plasma_applet_uim ${uim_SRCS})
 target_link_libraries(plasma_applet_uim ${KDE4_PLASMA_LIBS} ${KDE4_KDEUI_LIBS}
